[2024-09-06T07:32:44.339+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-06T07:32:44.368+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline.upload_file_to_s3 manual__2024-09-06T07:32:35.180791+00:00 [queued]>
[2024-09-06T07:32:44.376+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline.upload_file_to_s3 manual__2024-09-06T07:32:35.180791+00:00 [queued]>
[2024-09-06T07:32:44.376+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-09-06T07:32:44.393+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_file_to_s3> on 2024-09-06 07:32:35.180791+00:00
[2024-09-06T07:32:44.403+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=376) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-06T07:32:44.404+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline', 'upload_file_to_s3', 'manual__2024-09-06T07:32:35.180791+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/newspipeline.py', '--cfg-path', '/tmp/tmpapvc4hy4']
[2024-09-06T07:32:44.405+0000] {standard_task_runner.py:63} INFO - Started process 377 to run task
[2024-09-06T07:32:44.405+0000] {standard_task_runner.py:91} INFO - Job 15: Subtask upload_file_to_s3
[2024-09-06T07:32:44.473+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_data_pipeline.upload_file_to_s3 manual__2024-09-06T07:32:35.180791+00:00 [running]> on host ca47fb49acc7
[2024-09-06T07:32:44.565+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline' AIRFLOW_CTX_TASK_ID='upload_file_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-06T07:32:35.180791+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-06T07:32:35.180791+00:00'
[2024-09-06T07:32:44.566+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-06T07:32:46.643+0000] {newspipeline.py:139} INFO - File uploaded successfully to ai-news-pipeline/ai_news_articles.csv
[2024-09-06T07:32:46.645+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-06T07:32:46.646+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-06T07:32:46.655+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline, task_id=upload_file_to_s3, run_id=manual__2024-09-06T07:32:35.180791+00:00, execution_date=20240906T073235, start_date=20240906T073244, end_date=20240906T073246
[2024-09-06T07:32:46.695+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-06T07:32:52.008+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-06T07:32:52.295+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
